api/src/main/java/org/apache/cloudstack/api/command/user/network/CreateNetworkCmd.java

```diff
@@ -48,7 +49,7 @@
 
 @APICommand(name = "createNetwork", description = "Creates a network", responseObject = NetworkResponse.class, responseView = ResponseView.Restricted, entityType = {Network.class},
         requestHasSensitiveInfo = false, responseHasSensitiveInfo = false)
-public class CreateNetworkCmd extends BaseCmd implements UserCmd {
+public class CreateNetworkCmd extends BaseAsyncCmd implements UserCmd {
```

Comment:
@Pearl1594 can you start a discussion thread on dev@/users@ we want to see if it causes any backward compatibility issues for users

Reply:
+1